spec/models/manageiq/providers/microsoft/infra_manager/refresher_spec.rb

```diff
@@ -50,14 +50,14 @@
   end
 
   it "Classic same as graph refresh" do
-    stub_settings_merge(:ems_refresh => {:scvmm => {:inventory_object_refresh => false}})
-    EmsRefresh.refresh(@ems)
-    inventory_after_classic_refresh = serialize_inventory
-
```

Comment:
Running classic refresh first was covering up some cases where graph refresh wasn't setting some attribute and it was passing as a false positive